field-agent-ui/src/components/DeleteAgent.js

```diff
@@ -0,0 +1,120 @@
+import {useContext, useEffect, useState} from "react";
+import AuthContext from "../AuthContext";
+import { Link, useHistory, useParams } from 'react-router-dom';
+
+function DeleteAgent() {
+    const [firstName, setFirstName] = useState('');
+    const [middleName, setMiddleName] = useState('');
+    const [lastName, setLastName] = useState('');
+    const [dob, setDob] = useState('');
+    const [heightInInches, setHeightInInches] = useState('');
+    const auth = useContext(AuthContext);
+    const [agents, setAgents] = useState([]);
+    const [errors, setErrors] = useState([]);
+    const { id } = useParams();
+    const history = useHistory();
+
+    const getAgentToDelete = (token) => {
+        const init = {
+            headers: {
+                'Authorization': `Bearer ${token}`
+            }
+        }
+
+        fetch(`http://localhost:8080/api/agent/${id}`, init)
+            // Response object
+            .then(response => {
+                if (response.status === 404) {
+                    return Promise.reject(`Received 404 Not Found for Agent ID: ${id}`);
+                }
+                return response.json();
+            })
+            .then(data => {
+                setFirstName(data.firstName);
+                setMiddleName(data.middleName);
+                setLastName(data.lastName);
+                setDob(data.dob);
+                setHeightInInches(data.heightInInches);
+            })
+            .catch(error => {
+                console.log(error);
+            });
+    }
+
+    const getAgents = (token) => {
+        /*
+        GET http://localhost:8080/api/agents HTTP/1.1
+        Authorization: Bearer {{token}}
+        */
+
+        const init = {
+            headers: {
+                'Authorization': `Bearer ${token}`
+            }
+        }
+
+        // loading initial data for our component
+        fetch('http://localhost:8080/api/agent', init)
+            .then(response => response.json())
+            .then(data => setAgents(data))
+            .catch(error => console.log(error));
+    };
```

Comment:
Remove the `getAgents()` function from this component. The user will see the updated list of agents when you redirect them back to the Agents component.